src/main/java/uk/gov/companieshouse/filingmock/model/InsolvencyPractitioners.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.companieshouse.filingmock.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSetter;
+
+import java.util.List;
+
+/**
+ * Class to hold the data for the insolvency practitioners array of objects
+ */
+
+public class InsolvencyPractitioners {
+    
+    @JsonProperty("practitioners")
+    private List<PractitionerWrapper> practitioners;
+
+
+    public String getPractitionerOnePostcode() {
+        if (practitioners == null || practitioners.isEmpty()) {
+            return null;
+        }
+        return practitioners.get(0).address.getPostalCode();
+    }
+
+}
+
+class PractitionerWrapper {
```

Comment:
Discussed and explained that although there is only currently an address field, the wrapper is indeed a practitioner wrapper as it could in principle be extended to reflect other practitioner fields e.g. ip_code, should the mocking ever wish to use those values in its logic. The brevity of the classes is because current mocking depends only on practitioner postcode - but they could be fleshed out as required.